recipes/pyscrap/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ "${PY_VER}" =~ 3 ]]
+then
+    2to3 -w -n setup.py
```

Comment:
@conda-forge/help-python -- is there any policy regarding applying `2to3` to get python3 compatible builds?

Reply:
Thanks for the feedback @synapticarbors ! I fixed the recipe as suggested. I still left  the "2to3" part though - but I can remove it and limit the recipe to only py27 if needed.

Reply:
@synapticarbors , are there any updates on this?